teuthology/schedule.py

```diff
@@ -94,23 +93,18 @@ def schedule_job(job_config, num=1, report_status=True):
     :param num:      The number of times to schedule the job
     """
     num = int(num)
-    job = yaml.safe_dump(job_config)
-    tube = job_config.pop('tube')
-    beanstalk = teuthology.beanstalk.connect()
-    beanstalk.use(tube)
+    '''
+    Add 'machine_type' queue to DB here.
+    '''
+    report.create_machine_type_queue(job_config['machine_type'])
     while num > 0:
-        jid = beanstalk.put(
-            job,
-            ttr=60 * 60 * 24,
-            priority=job_config['priority'],
-        )
-        print('Job scheduled with name {name} and ID {jid}'.format(
-            name=job_config['name'], jid=jid))
```

Comment:
Can we just rename schedule_job to beanstalk_schedule_job and for paddles create a separate paddles_schedule_job?